libs/api/domains/rsk-company-info/src/lib/rsk-company-info-module.ts

```diff
@@ -0,0 +1,37 @@
+import { DynamicModule, Module } from '@nestjs/common'
+import {
+  RskCompanyInfoServiceOptions,
+  CompanyApi,
+  Configuration as RestConfiguration,
+} from '@island.is/clients/rsk-company-info'
+import { RskCompanyInfoResolver } from './graphql/rsk-company-info.resolver'
+import { RskCompanyInfoService } from './graphql/rsk-company-info.service'
+
+@Module({})
+export class RskCompanyInfoModule {
+  static register(config: RskCompanyInfoServiceOptions): DynamicModule {
+    const { xRoadBaseUrl, xRoadProviderId } = config
+    const RSK_COMPANY_BASE_PATH = `${xRoadBaseUrl}/r1/${xRoadProviderId}/`
+    return {
+      module: RskCompanyInfoModule,
+      providers: [
+        RskCompanyInfoResolver,
+        RskCompanyInfoService,
+        {
+          provide: CompanyApi,
+          useFactory: () =>
+            new CompanyApi(
+              new RestConfiguration({
+                fetchApi: fetch,
```

Comment:
Please use [createEnhancedFetch](https://github.com/island-is/island.is/blob/2fdf9c5f2aaac1607a0e8a1a1ab363f2e326e1e0/libs/clients/middlewares/README.md)